src/astra_driver.cpp

```diff
@@ -79,6 +79,10 @@ AstraDriver::AstraDriver(rclcpp::node::Node::SharedPtr& n, rclcpp::node::Node::S
   z_scaling_ = 1.0;
   z_offset_mm_ = 0;
 
+  AstraVideoMode color_video_mode{width, height, framerate, PIXEL_FORMAT_RGB888};
+
+  setColorVideoMode(color_video_mode);
```

Comment:
Man, I thought I tried this, but it didn't seem to work (it was hard coded and not as nice as this pr, but I thought it still should have worked...)